### DISCUSSION TITLE
Change default "main" color to .700

### DISCUSSION THREAD

**Discussion by Philippe-Tic** [chakra-ui/chakra-ui#2065] [General]
Hi there! 
I've got an issue and I don't know if there is a simple solution for this.
I'm on a project with some accessibility constraints and I've got a graphical charter that I must use.

My problem is that I have a color variation that I would like to use, but I would like to work with my brand.700 as my primary color. (for all the component, as Button to have a good contrast.). 

Is there any way to change this default .500 colors ?

Also, I am on Chakra-UI V1. So I saw that I can change all the components properties in my theme with ma variant. It is a solution but I would like to know if there is something simpler :) 

Thx a lot for the answser !

**Comment by chasinhues** [comment]
@PhilippeBear sorry for the late reply! 

Unfortunately due to the way the theme works, a one-time setting to change the default colours across all components isn't really possible.

If you check out the [components in the theme package](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components), you'll notice that a lot of them use multiple different colour hues (100/200/500/900 & more). 

You would have to make your desired changes for each component in the theme.
